modules/ixBidAdapter.js

```diff
@@ -99,8 +100,46 @@ const VIDEO_PARAMS_ALLOW_LIST = [
   'playerSize', 'w', 'h'
 ];
 const LOCAL_STORAGE_KEY = 'ixdiag';
+export const LOCAL_STORAGE_FEATURE_TOGGLES_KEY = `${BIDDER_CODE}_features`;
 let hasRegisteredHandler = false;
 export const storage = getStorageManager({gvlid: GLOBAL_VENDOR_ID, bidderCode: BIDDER_CODE});
+export const FEATURE_TOGGLES = {
+  featureToggles: {},
+  isFeatureEnabled: function(ft) {
+    return deepAccess(this.featureToggles, `features.${ft}.activated`)
+  },
+  getFeatureToggles: function() {
+    if (storage.localStorageIsEnabled()) {
+      const parsedToggles = safeJSONParse(storage.getDataFromLocalStorage(LOCAL_STORAGE_FEATURE_TOGGLES_KEY));
+      if (deepAccess(parsedToggles, 'expiry') && parsedToggles.expiry >= new Date().getTime()) {
+        this.featureToggles = parsedToggles
+      } else {
+        this.clearFeatureToggles();
+      }
+    }
+  },
+  setFeatureToggles: function(serverResponse) {
+    const responseBody = serverResponse.body;
+    const expiryTime = new Date();
+    const toggles = deepAccess(responseBody, 'ext.features');
+
+    if (toggles) {
+      this.featureToggles = {
+        expiry: expiryTime.setHours(expiryTime.getHours() + 1),
+        features: toggles
+      }
+      if (storage.localStorageIsEnabled()) {
+        storage.setDataInLocalStorage(LOCAL_STORAGE_FEATURE_TOGGLES_KEY, JSON.stringify(this.featureToggles));
```

Comment:
Why is it useful to read these values from local storage instead of relying on configuration

Reply:
enabling post is dependent on our server response for the moment. We are reading from an internal config and local storage when available. The advantage of reading from local storage is we can capture requests on page load rather than waiting for a response.

Reply:
@patmmccann let me know if you have any additional questions.

Reply:
setting ix_features: {"expiry":<TIMESTAMP>,"features":{"pbjs_enable_post":{"activated":true}}}

in local storage and giving a bidder an override to enable post method seems to be a bizarre workflow for a publisher and also seems undocumented. I feel like I must be missing something here. Why couldn't the publisher opt into IX post mode via a more standard method?

Reply:
@patmmccann 

We won't be relying on local storage to activate POST. Local storage access will allow us to generate POST requests on page load but this is a dependency and more of a nice to have. 

If reading from local storage is not allowed then POST will be limited to ad refreshes on the page. In a future PR, we will be moving completely to POST. The intention here is to roll out POST within our adapter gradually.

Reply:
This is an unsatisfying reason; we'll discuss in committee. I recommend you plan on this not being approved.